http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/service/ProxyService.scala

```diff
@@ -161,91 +166,95 @@ class ProxyService(
       } else IO.unit
     } yield ()
 
-  def proxyLocalize(userInfo: UserInfo, googleProject: GoogleProject, runtimeName: RuntimeName, request: HttpRequest)(
-    implicit ev: ApplicativeAsk[IO, TraceId]
-  ): IO[HttpResponse] =
-    for {
-      _ <- authCheck(userInfo, googleProject, runtimeName, SyncDataToRuntime)
-      now <- timer.clock.realTime(TimeUnit.MILLISECONDS)
-      r <- proxyInternal(googleProject, runtimeName, request, Instant.ofEpochMilli(now))
-    } yield r
-
   def invalidateAccessToken(token: String): IO[Unit] =
     blocker.blockOn(IO(googleTokenCache.invalidate(token)))
 
-  /**
-   * Entry point to this class. Given a google project, cluster name, and HTTP request,
-   * looks up the notebook server IP and proxies the HTTP request to the notebook server.
-   * Returns NotFound if a notebook server IP could not be found for the project/cluster name.
-   * @param userInfo the current user
-   * @param googleProject the Google project
-   * @param runtimeName the cluster name
-   * @param request the HTTP request to proxy
-   * @return HttpResponse future representing the proxied response, or NotFound if a notebook
-   *         server IP could not be found.
-   */
   def proxyRequest(userInfo: UserInfo, googleProject: GoogleProject, runtimeName: RuntimeName, request: HttpRequest)(
     implicit ev: ApplicativeAsk[IO, TraceId]
   ): IO[HttpResponse] =
     for {
       _ <- authCheck(userInfo, googleProject, runtimeName, ConnectToRuntime)
-      now <- timer.clock.realTime(TimeUnit.MILLISECONDS)
-      r <- proxyInternal(googleProject, runtimeName, request, Instant.ofEpochMilli(now))
+      now <- nowInstant
```

Comment:
can probably require `implicit ev: ApplicativeAsk[IO, AppContext]` instead of using `nowInstant`